docs/usage.rst

```diff
@@ -51,3 +51,77 @@ without reloading page.
   <script type="{% js_type_for_cookie_consent request "social" "*:.google.com" %}" data-varname="social">
     alert("Social cookie accepted");
   </script>
+
+
+Asking users for cookie consent in templates
+--------------------------------------------
+
+``django-cookie-consent`` can website visitors a cookie consent message. This
+message informs users that the website uses cookies and requests their consent
+to store them. The script responsible for displaying the message is
+``cookiebar.js``. You can see an example of its usage in the testapp.
+ 
+In order to display the cookie consent message on your website:
+
+1. Load the ``cookiebar.js`` script in your HTML template. You can do this by
+   adding the following line to the ``<head>`` section of your template:
+
+.. code-block:: html
+  <script type="text/javascript" src={% static "cookie_consent/cookiebar.js" %}></script>
```

Comment:
```suggestion
.. code-block:: html

  <script type="text/javascript" src="{% static 'cookie_consent/cookiebar.js' %}"></script>
```
I think this needs a blank line to properly display, and I'd also recommend properly quoting in the markup